test/e2e_node/eviction_test.go

```diff
@@ -289,50 +289,50 @@ var _ = framework.KubeDescribe("PriorityMemoryEvictionOrdering [Slow] [Serial] [
 	highPriorityClassName := f.BaseName + "-high-priority"
 	highPriority := int32(999999999)
 
-	BeforeEach(func() {
-		_, err := f.ClientSet.SchedulingV1beta1().PriorityClasses().Create(&schedulerapi.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: highPriorityClassName}, Value: highPriority})
-		Expect(err == nil || errors.IsAlreadyExists(err)).To(BeTrue())
-	})
-
-	AfterEach(func() {
-		err := f.ClientSet.SchedulingV1beta1().PriorityClasses().Delete(highPriorityClassName, &metav1.DeleteOptions{})
-		Expect(err).NotTo(HaveOccurred())
-	})
-
 	Context(fmt.Sprintf(testContextFmt, expectedNodeCondition), func() {
-		tempSetCurrentKubeletConfig(f, func(initialConfig *kubeletconfig.KubeletConfiguration) {
-			memoryConsumed := resource.MustParse("600Mi")
-			summary := eventuallyGetSummary()
-			availableBytes := *(summary.Node.Memory.AvailableBytes)
-			if availableBytes <= uint64(memoryConsumed.Value()) {
-				framework.Skipf("Too little memory free on the host for the PriorityMemoryEvictionOrdering test to run")
+		BeforeEach(func() {
+			_, err := f.ClientSet.SchedulingV1beta1().PriorityClasses().Create(&schedulerapi.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: highPriorityClassName}, Value: highPriority})
+			Expect(err == nil || errors.IsAlreadyExists(err)).To(BeTrue())
+		})
+		AfterEach(func() {
+			err := f.ClientSet.SchedulingV1beta1().PriorityClasses().Delete(highPriorityClassName, &metav1.DeleteOptions{})
+			Expect(err).NotTo(HaveOccurred())
+		})
+		Context("", func() {
```

Comment:
Why not just use `It(`?

Reply:
Turns out I don't need the extra context... I just moved this into the existing context.